app/views/inbox/_entry.html.haml

```diff
@@ -40,9 +40,9 @@
           = render "shared/format_link"
     .card-footer.d-none{ id: "ib-options-#{i.id}" }
       %h4= t(".sharing.heading")
-      - if current_user.services.count.positive?
+      - if @services.count.positive?
```

Comment:
Please pass the `@services` into the entry partial so you can use it as `services` and avoid the haml-lint warning.